docs/concepts/economics/index.md

```diff
@@ -55,7 +55,7 @@ Fixed pricing means the gas costs are determined using a cost table, and transac
 
 When fees are eliminated, the block proposer receives validator credits instead of transaction fees. These credits contribute to the validator's total weight, determining their chances of winning a slot in the next era. Validators get rewards for proposing a block and creating and publishing finality signatures. In essence, gas/balance holds ensure that the network still compensates validators for their computations.
 
-The fee elimination model is different than the refund model introduced on Mainnet with release 1.5.6 and has replaced the refund behavior. Since all these behaviors are configurable, [private networks](../../operators/setup-network/create-private.md) can set their fee, refund, and pricing strategies.
+The fee elimination model will be different from the refund model introduced on Mainnet with release 1.5.6. Since all these behaviors are configurable, [private networks](../../operators/setup-network/create-private.md) can set their fee, refund, and pricing strategies.
```

Comment:
I thought we were removing/hiding the private network info? If so, this reference should go too.

Reply:
Thank you, removed the private network reference here.